src/ImuSensor.cc

```diff
@@ -116,6 +116,18 @@ bool ImuSensor::Load(const sdf::Sensor &_sdf)
     return false;
   }
 
+  // Set orientation reference frame
+  // TODO(adityapande-1995) : Add support for named frames like
+  // ENU using ign-gazebo
+  if (_sdf.ImuSensor()->Localization() == "CUSTOM") {
```

Comment:
nit: it's not part of our linters, but the ignition code style places `{` on the following line (see code above). If you don't mind following this pattern throughout this PR, it will help with our code consistency

Reply:
Done